tests/e2e/pageObjects/index.ts

```diff
@@ -37,5 +38,6 @@ export {
     PubSubPage,
     SlowLogPage,
     NotificationPage,
-    OnboardingPage
+    OnboardingPage,
+    InsightsPage
```

Comment:
it's better add comma after the last element to do not have 2 lined edited in git diff